### DISCUSSION TITLE
Option to hide more in profile when set to private

### DISCUSSION THREAD

**Discussion by emiliebarnard** [community/community#65485] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

I would appreciate an option to hide additional information, such as bio, place of work, and location, when my profile is set to private. The ideal situation would be for only my mutual followers to be able to see this content, or an option for me to turn that on.

This is incredibly important for employees in public positions prone to harassment.

**Comment by hades255** [comment]
Currently, GitHub does not provide options to selectively hide certain information from your profile for private accounts. However, GitHub does allow you to choose which information is displayed publicly on your profile, regardless of whether your account is public or private.

To control the visibility of your profile information, you can go to your GitHub profile settings. There you can choose which personal information, such as bio, company, and location, is shown publicly or kept private. Unfortunately, it's not possible to limit this visibility only to mutual followers. The options are either to display the information publicly or keep it private for all viewers.

If you have privacy concerns, make sure to adjust your settings accordingly and share only what you are comfortable with publicly displaying on your profile. Keep an eye on future updates from GitHub, as they might introduce new features that align with your request.
